Yank/commands/script.py

```diff
@@ -25,14 +25,17 @@
 YANK script
 
 Usage:
-  yank script (-y=FILEPATH | --yaml=FILEPATH)
+  yank script (-y=FILEPATH | --yaml=FILEPATH) [--platform=PLATFORM]
```

Comment:
Why is this its own option instead of just part of the script? Having a redundant way to do the same action when not meant as a backup will confuse people.